packages/apollo-gateway/src/index.ts

```diff
@@ -84,6 +104,15 @@ export class ApolloGateway implements GraphQLService {
       this.createSchema(config.localServiceList);
     }
 
+    if (isHostedConfig(config)) {
+      const apiKey = config.apiKey || process.env['ENGINE_API_KEY'];
+      if (!apiKey) {
+        throw new Error(
+          'The gateway requires either a serviceList, localServiceList, or apiKey to be provided in the config, or ENGINE_API_KEY to be defined in the environment',
```

Comment:
Main suggestion here is to call it Apollo Gateway for clarity since, presumably, there are might be other gateways involved in implementor's infrastructure.  I then went on to make a couple tweaks, including quotes around the relevant symbols, but also a period at the end of the sentence.  (It is a sentence, after all.)

```suggestion
          'Apollo Gateway requires either a `serviceList`, `localServiceList`, or `apiKey` to be provided in the config, or `ENGINE_API_KEY` to be defined in the environment.',
```